db/seeds.rb

```diff
@@ -5,3 +5,18 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Neighborhood.create(
+    [
+    { name: 'Chinatown' }, { name: 'Back Bay' }, { name: 'Financial District' },
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.